examples/websocket_server.rs

```diff
@@ -239,28 +246,16 @@ async fn run(executor: TaskExecutor) {
             // TODO this needs to be made into a proper iter later, instead of a vec and reconstruction
             for e in entities.iter_mut() {
                 // bounce off the walls if they're exceeding the boundaries
-                if e.x - e.radius <= MIN_X {
-                    e.vx = e.vx * -1.0;
-                    e.x = 0.1f64 + e.radius;
-                } else if e.x + e.radius >= MAX_X {
+                if e.x - e.radius <= MIN_X || e.x + e.radius >= MAX_X {
                     e.vx = e.vx * -1.0;
-                    e.x = 19.9f64 - e.radius;
                 }
 
-                if e.y - e.radius <= MIN_Y {
-                    e.vy = e.vy * -1.0;
-                    e.y = 0.01f64 + e.radius;
-                } else if e.y + e.radius >= MAX_Y {
+                if e.y - e.radius <= MIN_Y || e.y + e.radius >= MAX_Y {
                     e.vy = e.vy * -1.0;
-                    e.y = 19.9f64 - e.radius;
                 }
 
-                if e.z - e.radius <= MIN_Z {
+                if e.z - e.radius <= MIN_Z || e.z + e.radius >= MAX_Z {
```

Comment:
This works in 3d. The only reason I had it like this before was because if they ever leave the bounds box at all in ChartJS, they disappear, so I had to reset the position so they're fully contained. I don't see this being a problem in ThreeJS.